docs/mindsdb-docs/docs/sql/tutorials/house-sales-forecasting.md

```diff
@@ -1,60 +1,158 @@
-# Forecast quarterly house sales using MindsDB
+# Forecasting Quarterly House Sales with MindsDB
 
 ## Introduction
 
-In this short example we will produce forecasts for a multivariate time series.
+In this tutorial, we'll create, train, and query a machine learning model, which, in MindsDB language, is an `AI Table` or a `predictor`. We aim to produce forecasts for a multivariate time series.
```

Comment:
Maybe we could also explain here what we want to actually predict, which is house sales 

Something more like: 
We aim to produce sales forecast for houses using a multivariate time series strategy

Reply:
Done.